examples/xscope/host/src/host.c

```diff
@@ -1,70 +1,72 @@
-// Copyright 2016-2021 XMOS LIMITED.
+// Copyright 2016-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 #include <stdio.h>
-#include <string.h>
-#include <signal.h>
 #include <stdlib.h>
-#include <stdint.h>
-#include "xscope_endpoint.h"
 #include "control_host.h"
-#include "signals.h"
+#include "util.h"
 #include "resource.h"
-#include "pause.h"
 
-void shutdown(void)
+int done = 0;
```

Comment:
I think the done variable is not used, it can be removed

Reply:
removed that variable in all the examples.